### ISSUE TITLE
Parser failure in @noble/curves

### ISSUE THREAD

**Issue by FabioPinheiro** [ScalablyTyped/Converter#558]
Just reporting a error:
```sbt
[error] Phase1ReadTypescript.scala:72 message Couldn't parse: Parse error at 80.48 ''>'' expected but identifier htf found [file => ../curves/abstract/bls.d.ts, thread => 741, project => fmgp-webapp, ms => 3824, phase => typescript, id => @noble/curves]com.olvind.logging.Logger$LoggedException: Couldn't parse: Parse error at 80.48 ''>'' expected but identifier htf found
[error] 	at com.olvind.logging.Logger$LoggingOps$.fatal$extension(Logger.scala:178)
[error] 	at org.scalablytyped.converter.internal.importer.Phase1ReadTypescript.$anonfun$apply$4(Phase1ReadTypescript.scala:72)
[error] 	at org.scalablytyped.converter.internal.importer.Lazy$$anon$1.get(Lazy.scala:30)
[error] 	at org.scalablytyped.converter.internal.importer.Phase1ReadTypescript.$anonfun$apply$22(Phase1ReadTypescript.scala:198)
[error] 	at org.scalablytyped.converter.internal.importer.Phase1ReadTypescript.apply(Phase1ReadTypescript.scala:198)
[error] 	at org.scalablytyped.converter.internal.importer.Phase1ReadTypescript.apply(Phase1ReadTypescript.scala:19)
[error] 	at org.scalablytyped.converter.internal.phases.PhaseRunner$.$anonfun$doNext$4(PhaseRunner.scala:71)
...
```

**Comment by FabioPinheiro** [comment]
Hum `@noble/curves` is only a dependency of what I wanted is there any work around?

**Comment by FabioPinheiro** [comment]
Forgot to reload sbt
`stIgnore += "@noble/curves"` does that

**Comment by oyvindberg** [comment]
Thanks for the report.

Always afraid to open a typescript file after ST parser failures to see what kind of shenanigans the TS people have been up to this time :D

**Comment by oyvindberg** [comment]
fixed in 2ad4b5000c3b48c09c4bd279c62eddffcad33bf7
